modules/ducktests/tests/ignitetest/tests/control_utility/consistency_test.py

```diff
@@ -108,3 +113,83 @@ def test_logging(self, ignite_version):
             pass
+
+    @cluster(num_nodes=4)
+    @ignite_versions(str(DEV_BRANCH))
+    @defaults(backups=[1], cache_count=[1], entry_count=[50_000], entry_size=[50_000], preloaders=[1])
+    def test_consistency_check_performance(self, ignite_version, backups, cache_count, entry_count, entry_size,
+                                           preloaders):
+        """
+        Tests time of performing consistency check and idle_verify utilities over the same data.
+        """
+
+        data_gen_params = DataGenerationParams(backups=backups, cache_count=cache_count, entry_count=entry_count,
+                                               entry_size=entry_size, preloaders=preloaders)
+
+        node_config = IgniteConfiguration(
+            version=IgniteVersion(ignite_version),
+            cluster_state="INACTIVE",
+            include_event_types=[EventType.EVT_CONSISTENCY_VIOLATION],
+            data_storage=DataStorageConfiguration(
+                max_wal_archive_size=2 * data_gen_params.data_region_max_size,
+                default=DataRegionConfiguration(
+                    persistence_enabled=True,
+                    max_size=data_gen_params.data_region_max_size
+                )
+            ),
+            metric_exporters={"org.apache.ignite.spi.metric.jmx.JmxMetricExporterSpi"}
+        )
+
+        ignites = IgniteService(self.test_context,
+                                config=node_config,
+                                num_nodes=self.test_context.available_cluster_size - data_gen_params.preloaders)
+        ignites.start()
+
+        control_utility = ControlUtility(ignites)
+
+        control_utility.activate()
+
+        preload_time = preload_data(
+            self.test_context,
+            ignites.config._replace(client_mode=True, discovery_spi=from_ignite_cluster(ignites)),
+            data_gen_params=data_gen_params)
+
+        start = current_millis()
+
+        control_utility.idle_verify()
+
+        finish = current_millis()
+
+        idle_verify_time = {
+            'start': start,
+            'finish': finish,
+            'time_to_run': (finish - start)
+        }
+
+        self.logger.info("Idle verify finished [time_to_run=" + str(idle_verify_time['time_to_run']) + ']')
+
+        start = current_millis()
+
+        for pi in range(0, 50):
+            # checking 20 partitions at a time
+            p = ','.join([str(x) for x in range(pi*20, (pi+1)*20)])
+
+            self.logger.debug('Running repair[p=' + p + ']')
+            # checking/repairing
+            control_utility.check_consistency(f"repair --cache test-cache-1 --strategy LWW --partitions " + p)
+
+        finish = current_millis()
+
+        check_consistency_time = {
+            'start': start,
+            'finish': finish,
+            'time_to_run': (finish - start)
+        }
+
+        self.logger.info("Check consistency finished [time_to_run=" + str(check_consistency_time['time_to_run']) + ']')
+
```

Comment:
We also should check that consistency check processed all the partitions/entries somehow.

Reply:
Check for lost partitions and correctness of the consistency utility is out of scope of this test.

Reply:
Check that specific partition was processed is added.